tests/extraction/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+mkdir -p run
+cd run
+
+../../../goblint --set ana.activated[+] extract-pthread ../00-sanity.c && mkdir -p ../../../pml-result &&  mv pml-result/pthread.pml ../../../pml-result/pthread.pml && spin -a ../../../pml-result/pthread.pml && cc -o pan pan.c && ./pan -a &> out.txt
```

Comment:
This is incredibly messy. Why do the test outputs have to be copied outside of the test tree instead of just copying the base model here once?

Also, this is precisely what #787 would be for.

Reply:
Yes, I agree it is a bit ugly, maybe I try copying the model into the appropriate folder.

I mean a cram test would depend on all the output of `spin` and it prints a whole ton of things, whereas this tries to identify just the verdict.